### ISSUE TITLE
[REPORT]: Disney's Treasure Planet

### ISSUE THREAD

**Issue by 10522** [ps2homebrew/Open-PS2-Loader-Compatibility-list#215]
### Checks

- [ ] I have checked if a report of the same game, in the same test environment, is already included
- [ ] I have tested all features from the feature list before making this report

### gameplay

doesn't even boot

### Console model

SCPH-90008

### OPL version / revision

OPL Beta 1.2.0 1938

### title

Disney's Treasure Planet

### Game Region code (ID)

SCUS_971.46

### device used on test

USB

### format

ZSO

### media

DVD5

### features

- [ ] VMC
- [ ] IGR
- [ ] PADEMU
- [ ] GSM

### comments about this report

Black screen on startup. Compatibility modes do not help. I use redump image.

**Comment by INDRAPhilip** [comment]
This game has already been tested by me [Disney's Treasure Planet #144 ISO](https://github.com/ps2homebrew/Open-PS2-Loader-Compatibility-list/issues/144) and [Disney's Treasure Planet #145 ZSO](https://github.com/ps2homebrew/Open-PS2-Loader-Compatibility-list/issues/145) next time try to check if The Game hasn't already been tested.
@10522

**Comment by INDRAPhilip** [comment]
In this compatibility repertoire, aiming to test all features of the opl so much that the option there is "I have checked if a report of the same game, in the same test environment, is already included
and I have tested all features from the feature list before making this report" in the case of VMC, IGR, PADEMU and GSM from what I read from your comments in [842](https://github.com/ps2homebrew/Open-PS2-Loader/issues/842) you value compatibility, but to perform the tests and report in this repertoire, it is necessary to test all possible compatibility modes and features.

**Comment by 10522** [comment]
@INDRAPhilip, i specifically made this report to show that [your](https://github.com/ps2homebrew/Open-PS2-Loader-Compatibility-list/issues/145) report does not work for me and user @Phantasm90. I wonder what could be the problem, i seem to have everything like yours, except for the PS2 model.

How can I test all the features from the list if this game only shows a black screen?

By the way, can you please check these reports: [LEGO Racers 2](https://github.com/ps2homebrew/Open-PS2-Loader/issues/797), [Drome Racers](https://github.com/ps2homebrew/Open-PS2-Loader/issues/796). These games don't work for me, but they work for user @Phantasm90. We also seem to have everything the same, except for the PS2 model. Can the PS2 model affect game compatibility?

**Comment by INDRAPhilip** [comment]
This game seems to work fine with my compatible and currently available devices for my testing which are HDD, USB and MX4SIO except the GSM which does not work 'fine' on any of the iso or zso versions. 

For me the model of should not be that relevant at the time of testing because I have 70001 and 90010, and it shows that it works well this game on MX4SIO and USB, only not reported on these console models because my main being with one more input device is fat. It may have a black screen for me only if I enable GSM in the 'zso' version in the iso even works, but I do not recommend it.

In the future I am planning to buy an Ilink device and maybe another thin console with a working network input for testing. Later I will try to test this LEGO Racers 2 and Drome Racers to see if I can get the similar or different result from your @10522 and @Phantasm90, because in theory there shouldn't be these differences of games working or not on certain console models.

**Comment by INDRAPhilip** [comment]
@10522 I was able to test these games LEGO Racers 2 and Drome Racers, and they do not show what you report, there was no black screen or crash after some logo, I performed the tests on USB and MX4SIO with 128 GB netac SD with opl beta 1964 and opl beta 1917 BDM4 test consoles models 39001 and 90010.

**Comment by Phantasm90** [comment]
@INDRAPhilip Can you test SLUS_200.05.World Destruction League Thunder Tanks please? The game freezes at the select tank screen when i press X or the D-Pad

**Comment by INDRAPhilip** [comment]
@Phantasm90 I just tested this game, and it doesn't work in any way on the USB, in MX4SIO it gets to the part where you press start and select the tank to start the Game, but after that, the Game crashes after selecting the tank and pressing X on the HDD seems to work fine, it doesn't have these problems.

**Comment by 10522** [comment]
@INDRAPhilip, i've already tried everything I can, but I still don't have LEGO Racers 2 and Drome Racers working in the new OPL versions. Just like @Phantasm90 and me, Disney's Treasure Planet ZSO doesn't work, but it works for you. Perhaps only the OPL developers can find a solution to these problems.

@Phantasm90, you can try the PAL version, it doesn't freeze when you select a tank, but it freezes just before the start of the battle. NTSC version freezes in the same place when using Mode 4.

**Comment by INDRAPhilip** [comment]
@10522 Yes, possibly only they could solve this issue, I don't understand why the Game doesn't work for you, the solution is to wait. I will do a pull request posting observations about this game, SCUS_971,46.Disney's Treasure Planet.
